pixart.js

```diff
@@ -1 +1,31 @@
+// Like the A-Team, but with variables
 
+var setColorButton = document.getElementById('set-color');
+var inputField= document.getElementById('color-field');
+var brushElement = document.querySelector('.brush');
+//var style = document.getComputedStyle(brushElement, null);
+var brushColor = "#1B4370";
+console.log("brush color is "+brushColor)
+var bunchaSquaresMaaan = [];
+
+setColorButton.addEventListener( "click", setBrush);
+
+function setBrush() {
+  event.preventDefault();
+  brushElement.style.backgroundColor=inputField.value;
+}
+
+for (i=0; i < 8000; i++){
+  bunchaSquaresMaaan[i]=document.body.appendChild(document.createElement("div"));
```

Comment:
While it's not 'wrong', there's no need to use an array here. Instead you can just make a variable in the loop to hold the 'current square' and reference it for the following two lines:

```is
var currentSquare = document.createElement("div");
document.body.appendChild(currentSquare);

currentSquare.className="square";
currentSquare.addEventListener("mouseover", setSquareColor)
```